projects/1.19/assets/Yungs-Better-NetherFortresses/YungsBetterNetherFortresses-/lang/zh_cn.json

```
@@ -0,0 +1,6 @@
+{
+  "Use /locate structure betterfortresses:fortress instead!": "使用/locate结构更好的堡垒：堡垒代替",
+  "text.autoconfig.betterfortresses-fabric-1_19.title": "YUNG's Better Nether Fortresses",

```

```suggestion
  "text.autoconfig.betterfortresses-fabric-1_19.title": "YUNG的下界要塞优化",
```
这个不知道要不要改
是根据mcmod上对同系列模组的翻译来的